Dockerfile

```diff
@@ -1,13 +1,18 @@
-FROM codacy-flawfinder-base:latest
+FROM library/openjdk:8-jre-alpine
 
-LABEL maintainer="team@codacy.com"
+ARG toolVersion
 
-RUN adduser -u 2004 -D docker
-WORKDIR /opt/docker
+RUN \
+    apk add --no-cache bash python2 && \
```

Comment:
Can you check if we need python2 or we can upgrade to python3?

Reply:
Done